src/Controls/src/Core/Handlers/Items/iOS/SelectableItemsViewController.cs

```diff
@@ -55,10 +55,17 @@ internal void SelectItem(object selectedItem)
 			if (index.Section > -1 && index.Item > -1)
 			{
 				// Ensure the selected index is updated after the collection view's items generation is completed
-				CollectionView.PerformBatchUpdates(null, _ =>
+				if (!CollectionView.IsLoaded())
```

Comment:
The fix is the same in:
- SelectableItemsViewController.cs
- SelectableItemsViewController2.cs

an create a shared helper method?